website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -120,12 +130,94 @@ resource "vcd_nsxt_edgegateway" "nsxt-edge" {
 ```
 
 
+## Example Usage (Assigning NSX-T Edge Gateway to VDC group)
+
+```hcl
+data "vcd_nsxt_edge_cluster" "secondary" {
+  name = "edge-cluster-two"
+}
+
+data "vcd_external_network_v2" "nsxt-ext-net" {
+  name = "nsxt-edge"
+}
+
+data "vcd_vdc_group" "group1" {
+  name = "existing-group"
+}
+
+data "vcd_org_vdc" "vdc-1" {
+  name = "existing-group"
+}
+
+resource "vcd_nsxt_edgegateway" "nsxt-edge" {
+  org      = "my-org"
+  owner_id = data.vcd_vdc_group.group1.id
+
+  # VDC Group cannot be created directly in VDC Group - it 
+  # must originate in some VDC (belonging to destination VDC Group)
+  #
+  # `starting_vdc_id` field is optional. If only VDC Group 
+  # ID is specified in `owner_id` field - this resource 
+  # will will pick a random member
+  # VDC to precreate it and will move to destination
+  # VDC Group in a single apply cycle
```

Comment:
Nit: this whole comment block is strangely cut at different line lengths, which looks a bit slacky :)

Reply:
Ok, tried to tune it up a bit. The reason I make such short lines is that I learned over time that Terraform HCL examples have very narrow view area (even on very wide displays). This is show it looks now:


<img width="636" alt="image" src="https://user-images.githubusercontent.com/15804230/156230933-9c445861-81ec-4b83-a076-a4d8852769d9.png">